go.mod

```diff
@@ -1,18 +1,20 @@
 module github.com/coder/envbuilder
 
-go 1.22
-
-toolchain go1.22.3
+go 1.22.4
 
 // There are a few options we need added to Kaniko!
 // See: https://github.com/GoogleContainerTools/kaniko/compare/main...coder:kaniko:main
 replace github.com/GoogleContainerTools/kaniko => github.com/coder/kaniko v0.0.0-20240624091120-7208a49f5b15
 
+// Required to import codersdk due to gvisor dependency.
+replace tailscale.com => github.com/coder/tailscale v1.1.1-0.20240702054557-aa558fbe5374
```

Comment:
note: required to import codersdk

Reply:
Is the commit significant, or is it just used by virtue of being the tip of `main`?

Reply:
same commit as from `coder/coder` repo as imported